specs/pvs/corla.pvs

```diff
@@ -981,6 +999,8 @@ BEGIN
   IMPORTING elections,
             FAFSSL
 
+%%% Audit board members are members of the public who come from different political parties. According to 
+
```

Comment:
According to CDOS via email, from what I recall.  We'll have to probably canonicalize parties or add a couple of buttons to represent D/R to ensure that this isn't screwed up.  Alternatively, we just make the statement and leave it on the audit boards and the SOS to make the determination.  (I like the latter solution better.  We'll ask CDOS how they want it.  I'll file an issue with the appropriate labels.)